bootcamp-hub/values.yaml

```diff
@@ -0,0 +1,54 @@
+jupyterhub:
+  hub:
+    baseUrl: /bootcamp-hub/
+    annotations:
+      prometheus.io/scrape: "true"
+      # this needs to start with the value of `hub.baseUrl`
+      prometheus.io/path: /bootcamp-hub/hub/metrics
+    extraConfig:
+      auth: |
+        c.JupyterHub.authenticator_class = 'hashauthenticator.HashAuthenticator'
+      admin: |
+        c.Authenticator.admin_users = {'leah-admin', 'tim-admin'}
+        c.JupyterHub.admin_access = True
+    image:
+      # tag will be set by travis on deployment
+      name: earthlabhubops/ea-k8s-hub-wshub
+      tag: set-on-deployment
+  singleuser:
+    image:
+      # tag will be set by travis on deployment
+      name: earthlabhubops/ea-k8s-user-wshub
```

Comment:
This configures the environment the user finds themselves in. So for example what libraries are installed. Same convention about naming as above for the hub image apply here.

Reply:
got it. see note about about using dashes OR NOT in the hub names...